rs/dogecoin/ckdoge/minter/BUILD.bazel

```diff
@@ -0,0 +1,101 @@
+load("@rules_rust//rust:defs.bzl", "rust_library", "rust_test")
+load("//bazel:canisters.bzl", "rust_canister")
+load("//bazel:defs.bzl", "rust_ic_test")
+
+package(default_visibility = ["//rs/dogecoin/ckdoge/test_utils:__subpackages__"])
+
+rust_library(
+    name = "ckdoge_minter_lib",
+    srcs = glob(
+        ["src/**"],
+        exclude = ["src/main.rs"],
+    ),
+    crate_features = [],
+    crate_name = "ic_ckdoge_minter",
+    proc_macro_deps = [
+        # Keep sorted.
+        "@crate_index//:async-trait",
+    ],
+    version = "0.1.0",
+    deps = [
+        # Keep sorted.
+        "//packages/icrc-ledger-types:icrc_ledger_types",
+        "//rs/bitcoin/ckbtc/minter:ckbtc_minter_lib",
+        "@crate_index//:candid",
+        "@crate_index//:ic-cdk",
+        "@crate_index//:serde",
+    ],
+)
+
+[
+    rust_canister(
+        name = name,
+        srcs = ["src/main.rs"],
+        compile_data = [":ckdoge_minter.did"],
+        crate_features = features,
+        crate_name = "ic_ckdoge_minter_canister",
+        proc_macro_deps = [
+            # Keep sorted.
+        ],
+        rustc_env = {
+            "CKDOGE_MINTER_DID_PATH": "$(execpath :ckdoge_minter.did)",
+        },
+        service_file = ":ckdoge_minter.did",
+        deps = [
+            # Keep sorted.
+            ":ckdoge_minter_lib",
+            "//rs/bitcoin/ckbtc/minter:ckbtc_minter_lib",
+            "@crate_index//:candid",
+            "@crate_index//:ic-cdk",
+        ],
+    )
+    for (name, features) in [
+        # Production version without debug assertions.
+        (
+            "ckdoge_minter",
+            [],
+        ),
+        # Test version with internal consistency checks.
+        (
+            "ckdoge_minter_debug",
+            ["self_check"],
+        ),
+    ]
+]
+
+rust_test(
+    name = "ckdoge_candid_api_tests",
+    crate = ":_wasm_ckdoge_minter",
+    data = [":ckdoge_minter.did"],
+    env = {
+        "CARGO_MANIFEST_DIR": "rs/dogecoin/ckdoge/minter",
```

Comment:
Is the reason to use `CARGO_MANIFEST_DIR` to point to the `.did` file so that `cargo test` also works?

Otherwise it would be simpler to use a dedicated env var to point to the `.did` file directly like: 
```
"SERVICE_FILE_PATH": "$(rootpath :ckdoge_minter.did)"
```